scripts/migrate_score_override.py

```diff
@@ -29,12 +29,21 @@ def main():
                 # prefetch the score summary and resave it to maintain its original field values.
                 score_summary = ScoreSummary.objects.get(student_item=score.student_item)
 
-                # Update old override with submission from the score preceding it
-                submission = Score.objects.filter(
+                # Update old override with submission from the score preceding it.
+                # If none exists, look for it in the submissions table.
+                preceding_score = Score.objects.filter(
                     student_item=score.student_item,
                     created_at__lte=score.created_at,
                     submission__isnull=False,
-                ).order_by('-created_at')[:1].get().submission
+                ).order_by('-created_at')[:1]
+                if preceding_score.count():
+                    submission = preceding_score.get().submission
+                else:
+                    submission_qset = Submission.objects.filter(student_item=score.student_item)
+                    if submission_qset.count() > 1:
+                        raise Exception("MULTIPLE SUBMISSIONS FOR STUDENT_ITEM {}".format(score.student_item))
```

Comment:
Do we actually want to bail completely on the entire loop?
Should we log the failure and continue onto the next override?

Reply:
No since there can be multiple overrides for the same student_item, don't want to have it finish out of order